src/components/SearchBar/SearchBar.tsx

```diff
@@ -16,14 +21,29 @@ const SearchBar = (props: Props) => {
   const dispatch: DispatchType = useDispatch();
 
   const [searchValue, setSearchValue] = useState<string>("");
+  const [searchOption, setSearchOption] = useState<string>("1");
 
   useEffect(() => {
     dispatch(getAllJobsApi());
   }, [dispatch]);
 
   const handleSearchJob = () => {
+    if (!searchValue) {
```

Comment:
Please show the text when search result is empty.
Not use  sweetalert2

Reply:
Fixed